CHANGELOG.md

```diff
@@ -1,3 +1,77 @@
+CHANGES IN VERSION 1.19.0 (FROM 1.18.3)
+===================================
+
+Appium 1.19.0 is a minor release
```

Comment:
It makes sense to also mention about a possible memory leak, which has been fixed in the base driver: https://github.com/appium/appium-base-driver/pull/430
and about the fix related to unexpected server errors handling: https://github.com/appium/appium-base-driver/pull/432

Reply:
one more general feature: Webauthn routes have been added (they are part of W3C now); https://github.com/appium/appium-base-driver/pull/433

Reply:
https://github.com/appium/appium-base-driver/pull/424: We do not wait anymore for pending requests to finish from the client side, but force close them on the server side if the server is terminated